.gitignore

```diff
@@ -40,3 +40,4 @@ site/out/
 
 .vscode/*.log
 **/*.swp
+.coderv2/*
```

Comment:
❓ Should we call this `coderOSS`? That way it aligns with the name we use in the docs.

Reply:
I just called it that because we already store stuff in `~/.config/coderv2` -- should we rename this config dir as well?

Reply:
I like `coderOSS`!

Reply:
@jsjoeio is there an issue for this possible rename?

Reply:
Not that I know of - just something I thought of now. Would you like me to make one?

Reply:
Yes please, thanks!
It's probably eaiser to do sooner than later! Might be good to bring up in grooming next-next-week.

Reply:
Done! https://github.com/coder/coder/issues/2768